index.php

```diff
@@ -1,8 +1,13 @@
 <?php
 require_once 'DeckDAO.php';
 require_once 'functions.php';
-$deckDao = new DeckDao();
-$decks = $deckDao->fetchAll();
+$deckDAO = new DeckDAO();
+if (isset($_POST['name_of_deck'])) {
+    $newDeck = new Deck($_POST['name_of_deck'], $_POST['format'], $_POST['colourid'], $_POST['archetype']
```

Comment:
Worth adding some basic checks to make sure you actually have all the data - IE they haven't left fields empty

Reply:
Added something to check if it contains strings with length > 0.